misc/acrn-config/board_config/new_board_kconfig.py

```diff
@@ -145,4 +145,9 @@ def generate_file(config):
     print("CONFIG_HV_RAM_START={}".format(hex(hv_start_addr)), file=config)
     print("CONFIG_HV_RAM_SIZE={}".format(hex(hv_ram_size)), file=config)
 
+    cpu_core_num = len(board_cfg_lib.get_processor_info())
+    if cpu_core_num == 2:
```

Comment:
just 2? How about > 2?

Reply:
Currently, KATA VM is one optional VM in SDC scenario, when we the cpu number have 2 cpu cores, they will be assigned to SOS_VM and POST VM, and there is no available cpu for KATA.
But if the cpu core > 2, there is available cpu for KATA, so support KATA VM for sdc scenario if  more than 2 cpu cores.

Reply:
OK